vcx/libvcx/src/messages/mod.rs

```diff
@@ -850,11 +851,15 @@ fn parse_response_from_agency_v1(response: &Vec<u8>) -> VcxResult<Vec<A2AMessage
     let verkey = settings::get_config_value(settings::CONFIG_SDK_TO_REMOTE_VERKEY)?;
     let (_, data) = crypto::parse_msg(&verkey, &response)?;
     let bundle: Bundled<Vec<u8>> = bundle_from_u8(data)?;
-    bundle.bundled
+    let messages = bundle.bundled
         .iter()
-        .map(|msg| rmp_serde::from_slice(msg)
-            .map_err(|err| VcxError::from_msg(VcxErrorKind::InvalidJson, format!("Cannot deserialize response: {}", err))))
-        .collect::<VcxResult<Vec<A2AMessage>>>()
+        .map(|msg| {
+            debug!("AgencyComm Inbound V1:\n{}",  std::str::from_utf8(msg.as_slice()).unwrap_or("Agency response contained non-utf8 characters."));
```

Comment:
What is the reason to try to parse utf8 from MessagePack bytes? It's likely impossible, isn't it? Should we just trace the result of rmp_serde::from_slice instead?